edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/IDeviceScopeApiClient.cs

```diff
@@ -2,9 +2,12 @@
 namespace Microsoft.Azure.Devices.Edge.Hub.CloudProxy
 {
     using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Hub.Core;
 
     public interface IDeviceScopeApiClient
     {
+        IDeviceScopeApiClient CreateOnBehalfOfDeviceScopeClient(string deviceId);
+
         Task<ScopeResult> GetIdentitiesInScope();
```

Comment:
should we fix the naming as this method returns Task<ScopeResult>, Async suffix should be used.  Same for GetNext method.
```suggestion
        Task<ScopeResult> GetIdentitiesInScopeAsync();
```

Reply:
Yea good idea